templates/ingress.yaml

```diff
@@ -1 +1,37 @@
-# TODO: Research & add Ingress controller for exposing st2web service to public net (#6)
+{{- if .Values.ingress.enabled }}
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  {{- $name := printf "%s-%s" $.Release.Name .Values.ingress.name | trunc 63 | trimSuffix "-" | quote }}
+  name: {{ $name }}
+  labels:
+    app: {{ $name }}
+    chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
+    release: "{{ $.Release.Name }}"
+    heritage: "{{ $.Release.Service }}"
```

Comment:
Please follow the `labels` from the other K8s resources.
There are `tier`, `vendor` and `support` labels missing